go.mod

```diff
@@ -0,0 +1,72 @@
+	github.com/ajeddeloh/yaml v0.0.0-20170912190910-6b94386aeefd // indirect
+	github.com/appscode/jsonpatch v0.0.0-20190108182946-7c0e3b262f30 // indirect
+	github.com/coreos/container-linux-config-transpiler v0.9.0
+	github.com/coreos/go-semver v0.3.0 // indirect
+	github.com/coreos/go-systemd v0.0.0-20190321100706-95778dfbb74e // indirect
+	github.com/coreos/ignition v0.32.0 // indirect
+	github.com/go-logr/logr v0.1.0 // indirect
+	github.com/go-logr/zapr v0.1.1 // indirect
+	github.com/gobuffalo/envy v1.7.0 // indirect
+	github.com/gogo/protobuf v1.2.1 // indirect
+	github.com/golang/groupcache v0.0.0-20190129154638-5b532d6fd5ef // indirect
+	github.com/google/btree v1.0.0 // indirect
+	github.com/google/gofuzz v1.0.0 // indirect
+	github.com/google/uuid v1.1.1 // indirect
+	github.com/googleapis/gnostic v0.3.0 // indirect
+	github.com/gophercloud/gophercloud v0.0.0-20190615014256-8a9023050033
+	github.com/gophercloud/utils v0.0.0-20190527093828-25f1b77b8c03
+	github.com/gregjones/httpcache v0.0.0-20190611155906-901d90724c79 // indirect
+	github.com/grpc-ecosystem/grpc-gateway v1.9.1 // indirect
+	github.com/imdario/mergo v0.3.7 // indirect
+	github.com/markbates/inflect v1.0.4 // indirect
+	github.com/pborman/uuid v1.2.0 // indirect
+	github.com/peterbourgon/diskv v2.0.1+incompatible // indirect
+	github.com/pkg/errors v0.8.1
+	github.com/prometheus/client_golang v0.9.4 // indirect
+	github.com/rogpeppe/go-internal v1.3.0 // indirect
+	github.com/spf13/afero v1.2.2 // indirect
+	github.com/spf13/cobra v0.0.5 // indirect
+	github.com/vincent-petithory/dataurl v0.0.0-20160330182126-9a301d65acbb // indirect
+	go.uber.org/atomic v1.4.0 // indirect
+	go.uber.org/multierr v1.1.0 // indirect
+	go.uber.org/zap v1.10.0 // indirect
+	go4.org v0.0.0-20190313082347-94abd6928b1d // indirect
+	golang.org/x/crypto v0.0.0-20190611184440-5c40567a22f8 // indirect
+	golang.org/x/net v0.0.0-20190613194153-d28f0bde5980 // indirect
+	golang.org/x/sys v0.0.0-20190616124812-15dcb6c0061f // indirect
+	golang.org/x/time v0.0.0-20190308202827-9d24e82272b4 // indirect
+	golang.org/x/tools v0.0.0-20190614205625-5aca471b1d59 // indirect
+	google.golang.org/appengine v1.6.1 // indirect
+	google.golang.org/genproto v0.0.0-20190611190212-a7e196e89fd3 // indirect
+	google.golang.org/grpc v1.21.1 // indirect
+	gopkg.in/inf.v0 v0.9.1 // indirect
+	gopkg.in/yaml.v2 v2.2.2
+	k8s.io/api v0.0.0-20190612125737-db0771252981
+	k8s.io/apiextensions-apiserver v0.0.0-20190228180357-d002e88f6236 // indirect
+	k8s.io/apimachinery v0.0.0-20190612125636-6a5db36e93ad
+	k8s.io/client-go v0.0.0-20190228174230-b40b2a5939e4
+	k8s.io/cluster-bootstrap v0.0.0-20190612131323-aa3fd9f69a09
+	k8s.io/code-generator v0.0.0-20190612125529-c522cb6c26aa
+	k8s.io/component-base v0.0.0-20190615090122-7edc45fbd48e // indirect
+	k8s.io/gengo v0.0.0-20190327210449-e17681d19d3a // indirect
+	k8s.io/klog v0.3.3
+	k8s.io/kube-openapi v0.0.0-20190603182131-db7b694dc208 // indirect
+	sigs.k8s.io/cluster-api v0.1.4
```

Comment:
previously it's branch release-0.1 now it's a tag ? just want to make sure understand is correct

Reply:
Yes, The go.mod file generally use semantics versions which is introduced by tagging a revision in the underlying source repository. 
https://golang.org/cmd/go/#hdr-Pseudo_versions